neptune/internal/backends/hosted_neptune_backend.py

```diff
@@ -209,6 +216,50 @@ def _get_client_config(self, backend_client: SwaggerClient) -> ClientConfig:
             max_compatible_version=version.parse(max_compatible) if max_compatible else None
         )
 
+    @with_api_exceptions_handler
+    def create_checkpoint(self, notebook_id: uuid.UUID, jupyter_path: str, _file: Any = None) -> Optional[uuid.UUID]:
+        # TODO maybe write an endpoint in backend that accepts FormData to be swagger-compatible? shouldn't hurt much
+        #  and we would be able to call this normally
+        if _file is not None:
```

Comment:
You don't use it. Jupyter extension is responsible for uploading content.

Reply:
Actually, this is not true. Creating checkpoint (as in database entry) is separated from uploading it's content, which, in fact, is done by Jupyter